drkafka/src/test/java/com/pinterest/doctorkafka/KafkaClusterTest.java

```diff
@@ -59,6 +59,24 @@ void getAlternativeBrokersDuplicateReassignmentTest() throws Exception{
 
     ReplicaStatsManager.bytesInStats = testBytesInStats;
     ReplicaStatsManager.bytesOutStats = testBytesOutStats;
+  }
+
+  @AfterEach
+  void tearDownReplicaStatsManager(){
+    ReplicaStatsManager.bytesInStats = oldBytesInStats;
+    ReplicaStatsManager.bytesOutStats = oldBytesOutStats;
+  }
+
+  /**
+   * This test assures that getAlternativeBrokers does not return a many-to-one reassignment
+   * We want to make sure that multiple out-of-sync replicas of the same topic partition will not
+   * map to the same replacement broker, or duplicate reassigments will happen leading to invalid
+   * reassignment plans.
+   */
+  @Test
+  void getAlternativeBrokersDuplicateReassignmentTest() throws Exception{
```

Comment:
Please change test method names to start with `test` and follow camel case convention.